mito-ai/style/ChatTaskpane.css

```diff
@@ -69,6 +69,30 @@
     align-items: center;
 }
 
+.chat-messages {
+    flex: 1;
+    overflow-y: auto;
+    padding-bottom: 10px;
+}
+
+.chat-empty-message {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
```

Comment:
When the chat is empty, I'm still getting a vertical scrollbar. If we changed this from `heigh: 100%` to `flex-grow: 1` and gave the chat-input-container `flex-grow: 0` I think it might fix that issue?

If it takes more than 10 mins to fix this, let's just ignore it for now.

Reply:
Can't fix without breaking something else. Saving this for another time.